### PR TITLE
stub supports void type return value

### PR DESCRIPTION
**Pull request by fagai**
Hello!

* Type: bug fix | new feature
* Link to issue: https://github.com/phalcon/ide-stubs/pull/50
https://github.com/phalcon/zephir/issues/1977

In raising this pull request, I confirm the following:

- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [ ] I updated the CHANGELOG

Small description of change:
I added it because I didn't consider the case where the return value is void type.

Thanks

### CODE REVIEW

File: Library/Stubs/Generator.php [hunk 1]
```diff
@@ -348,7 +348,10 @@ protected function buildMethod(ClassMethod $method, bool $isInterface, string $i
             if ($supported > 1) {
                 $return = '';
             }
+        } elseif (version_compare(PHP_VERSION, '7.1.0', '>=') && $method->isVoid()) {
```
Comment (AlexNDRmac) [inline Library/Stubs/Generator.php:351]: Better to use comparison with constant, its faster:

```php
elseif (PHP_VERSION_ID >= 70100 && $method->isVoid())
```

**Review by AlexNDRmac — CHANGES_REQUESTED** [review]

File: Library/Stubs/Generator.php (hunk 1 above)
Comment (fagai) [reply Library/Stubs/Generator.php:351]: @AlexNDRmac 
thank you.
I adjusted it for other parts, but is this better?

**Comment by fagai** [comment]
https://github.com/phalcon/zephir/blob/development/Library/Stubs/Generator.php#L304-L309
Should I also remove this?

File: Library/Stubs/Generator.php (hunk 1 above)
Comment (AlexNDRmac) [reply Library/Stubs/Generator.php:351]: Oh... I see :) This should be refactored... I think we can leave it as is for now. Thanks :)

**Comment by AlexNDRmac** [comment]
> https://github.com/phalcon/zephir/blob/development/Library/Stubs/Generator.php#L304-L309
> Should I also remove this?

No, It should be removed with small refactoring later. Thank you.

**Comment by fagai** [comment]
okay. Thank you.

**Review by AlexNDRmac — APPROVED** [review]

**Comment by AlexNDRmac** [comment]
Thank you

**Pull request merged**